SygicMapsKit/Classes/NavigationModule/SYMKNavigationView.swift

```diff
@@ -87,10 +100,27 @@ public class SYMKNavigationView: UIView {
         }
     }
     
+    /// Setup instruction view for navigation module.
+    ///
+    /// - Parameter instructionView: view with navigating instructions.
+    public func setupInstructionView(_ instructionView: SYMKInstructionView?) {
+        self.instructionView?.removeFromSuperview()
+        guard let instructionView = instructionView else { return }
+        self.instructionView = instructionView
```

Comment:
toto hod este nad guard ak to je optional aby si to schoval ked dojde nil. takto ti tam ostane zbytocne referencia na view